benchmark/test/test_metrics.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Measurement tests for the control service benchmarks.
+"""
+
+from twisted.trial.unittest import TestCase
+from twisted.internet.task import Clock
+from twisted.internet.defer import maybeDeferred
+
+from benchmark.metrics import WallClock
+
+
+class WallClockTests(TestCase):
+
+    def test_wallclock(self):
+        """
+        Returns the difference in time from before and after call to
+        function.
+        """
+        clock = Clock()
+        wallclock = WallClock(clock=clock)
+        d = wallclock.measure(maybeDeferred, clock.advance, 1.23)
+        d.addCallback(self.assertEqual, 1.23)
```

Comment:
This needs to either `return d` or, if this isn't using the reactor, use `SynchronousTestCase`, which is faster & safer.